src/Hl7.Fhir.Specification.Tests/Source/ArtifactSummaryTests.cs

```diff
@@ -77,7 +77,7 @@ public void TestValueSetXmlSummary()
             // Common properties
             Assert.IsFalse(summary.IsBundleEntry);
             Assert.AreEqual(ResourceType.ValueSet.GetLiteral(), summary.ResourceTypeName);
-            Assert.IsTrue(summary.ResourceType == ResourceType.ValueSet);
+            Assert.IsTrue(summary.ResourceTypeName == ResourceType.ValueSet.GetLiteral());
```

Comment:
So, we have broken the public interface on this `summary` thing, right?  It's not used much, so this is ok.

Reply:
Yes, this breaks the interface. I will create a list of breaking changes.